test/gtest/ucp/ucp_test.cc

```diff
@@ -772,6 +772,7 @@ void ucp_test_base::entity::accept(int worker_index,
 
     status = ucp_ep_create(ucp_worker, &ep_params, &ep);
     if (status == UCS_ERR_UNREACHABLE) {
+        ++m_err_cntr;
```

Comment:
shall we increase m_unreachable_err_count?

Reply:
`m_unreachable_err_count` is a member of `test_ucp_sockaddr` class, not `entity`